libraries/rush-lib/src/api/ApprovedPackagesConfiguration.ts

```diff
@@ -2,7 +2,6 @@
 // See LICENSE in the project root for license information.
 
 import * as path from 'path';
```

Comment:
We had some discussion about whether `console.log()` would produce CRLF on Windows.  Empirical tests confirmed that it produces LF, and @iclanton  later confirmed this in the [Node.js source code](https://github.com/nodejs/node/blob/main/lib/internal/console/constructor.js#L287).

Thus it does seem that this PR should mostly eliminate CRLF from Rush's output. 👍